feat(changelog): containers-added-serverless-containers-api-v1-available 2026-04-22 (#6450)
scaleway/docs-content: changelog/april2026/2026-04-22-containers-added-serverless-containers-api-v1-available.mdx

---
title: Serverless Containers API v1 available
status: added
date: 2026-04-22
category: serverless
product: containers
---

> The Scaleway Serverless Containers API is now generally available as `v1` (previously `v1beta1` which still available). This stable release includes significant improvements and lays the foundation for upcoming advanced features.

Reviewer: ```suggestion
The Scaleway Serverless Containers API is now generally available as `v1` (previously `v1beta1`,  which is still available). This stable release includes significant improvements and lays the foundation for upcoming advanced features.
```

> Console, CLI, Terraform and other tools maintained by Scaleway will be gradually updated.

Reviewer: ```suggestion
Console, CLI, Terraform, and other tools maintained by Scaleway will be gradually updated.
```

> For details read [the dedicated v1 documentation](https://www.scaleway.com/en/docs/serverless-containers/reference-content/v1-migration-guide/).

Reviewer: ```suggestion
For details, see [the dedicated v1 documentation](/serverless-containers/reference-content/v1-migration-guide/).
```
